client/landing/jetpack-cloud/sections/pricing/jpcom-masterbar/style.scss

```diff
@@ -0,0 +1,283 @@
+@import '~@wordpress/base-styles/breakpoints';
+@import '~@wordpress/base-styles/mixins';
+
+/**
+ * These color come from Jetpack.com color pallette. We leave them as they are
+ * because we want to make this page resemble as closest as possible jetpack.com/pricing.
+ */
+$jpcom-primary: #00be28;
+$jpcom-gray: #87a6bc;
+$jpcom-gray-lighten20: lighten( $jpcom-gray, 20% ); // #c8d7e1
+$jpcom-gray-lighten30: lighten( $jpcom-gray, 30% ); // #e9eff3
+$jpcom-gray-darken30: darken( $jpcom-gray, 30% ); // #3d596d
```

Comment:
It's worth nothing that I couldn't find these colors in https://color-studio.blog/.